cpu/stm32/periph/flashpage.c

```diff
@@ -118,8 +126,19 @@ static void _erase_page(void *page_addr)
     pn = (uint8_t)page;
 #endif
     CNTRL_REG &= ~FLASH_CR_PNB;
+#if defined(CPU_FAM_STM32F4) || defined(CPU_FAM_STM32F7)
+    if (FLASHPAGE_DUAL_BANK && (pn > (FLASHPAGE_NUMOF / 2 - 1))) {
```

Comment:
```suggestion
    if (IS_USED(FLASHPAGE_DUAL_BANK) && (pn > (FLASHPAGE_NUMOF / 2 - 1))) {
```

Reply:
See above, `FLASH_CR_SNB_4` is only defined for the f7, f4 and f2 and not for the L4, wb, g4, g0 and l5